docs/howto/getting_started_markdown.md

```diff
@@ -146,6 +146,180 @@ development. For more information on Configuration Panel GUI
 development, go to [Alteryx UI
 SDK](https://help.alteryx.com/developer-help/ayx-ui-sdk).
 
+## Platform SDK Quickstart Guide
+
+#### Python SDK v2
+
+The steps below illustrate how to use the latest developer tools to
+create a custom plugin or tool for use in Alteryx. For more information
+about new SDKs and their usage within Alteryx, please visit [Platform
+SDK](../references/platform-sdk.md).
+
+### Requirements and Prerequisites
+
+Before you get started, make sure that these items are installed on your
+machine:
+
+-   Microsoft Windows 7 or Later (64-bit)
+-   Alteryx Designer Version 2021.2\*
+-   Python Version 3.8.5
+-   [pip](https://pypi.org/) (automatically installed with Python 3.8.5)
+-   [node](https://nodejs.org/en/download/) [14](https://nodejs.org/en/blog/release/v14.17.3/)
+-   [Git](https://git-scm.com/downloads)
+
+\*Alteryx Designer version 2021.4 or greater is required to use the Alteryx Python
+SDK v2.0.0.
+
+### CLI
+
+The AYX Plugin CLI provides you with a set of utilities to manage your
+plugin and is a great starting point for building a tool. AYX Python SDK
+installation is also covered in this section.
+
+Follow these steps to get started with the CLI:
+
+1.  Run this command to install the AYX Plugin CLI:\
+    \
+    `pip install ayx-plugin-cli`\
```

Comment:
These are already covered in Install at the top of the doc.

Reply:
removed this section and included link to the earlier part of the doc